### ISSUE TITLE
CTCLoss on Android

### ISSUE THREAD

**Issue by rimidalv** [keras-team/keras#9218]
Hello.

When I tried run model ( made with this code https://github.com/keras-team/keras/blob/master/examples/image_ocr.py ) on Android, I got this error in logcat. 

 java.lang.IllegalArgumentException: No OpKernel was registered to support Op 'CTCLoss' with these attrs.  Registered devices: [CPU], Registered kernels:
<no registered kernels>                                                                        
[[Node: ctc/CTCLoss = CTCLoss[ctc_merge_repeated=true, ignore_longer_outputs_than_inputs=false, preprocess_collapse_repeated=false](ctc/Log, ctc/ToInt64, ctc/ToInt32_2, ctc/Squeeze_1)]]
at org.tensorflow.Session.run(Native Method)

I think the error is from behind this one ctc/**ToInt64** , but I'm not sure.

And I changed in the code this lines:
```python
labels = Input(name='the_labels', shape=[10], dtype='float32')
input_length = Input(name='input_length', shape=[1], dtype='int32') #was int64
label_length = Input(name='label_length', shape=[1], dtype='int32') #was int64
```

How I can add ctc loss to execute on android ? Is it bug?
